packages/services/carbon.yml

```
@@ -0,0 +1,77 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/carbon-resources.schema.json
+library:
+  id: ibmdotcom-services
+  name: IBM.com Services
+  description: IBM.com ES6 service classes
+  noIndex: true
+  demoLinks:
+    - type: storybook
+      name: Storybook
+      action: link
+      url: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/services
+assets:
+  analytics-api:
+    name: AnalyticsAPI
+    description: Analytics API class with methods for firing analytics events on ibm.com.
```

```suggestion
    description: Analytics API class with methods for firing analytics events on IBM.com.
```

I know we're just copy/pasting existing descriptions, but we should capitalize ibm.com everywhere we display it on our website unless it's a URL. Maybe a find-all and capitalize after indexing all?